docs/src/GMB.md

```diff
@@ -0,0 +1,58 @@
+# GMB
+The dataset an extract from GMB corpus which is tagged, annotated, 
+and built specifically to train the classifier to predict named entities such as name, location, etc. 
+
+GMB is a fairly large corpus with a lot of annotations.
+Unfortunately, GMB is not perfect. It is not a gold standard corpus, meaning that it’s not completely human annotated and it’s not considered 100% correct. 
+The corpus is created by using already existed annotators and then corrected by humans where needed.
+
```

Comment:
Can we link to the source of the dataset and also explain and/or link to the list of tags?

Reply:
Yeah I will add it

Reply:
Should i add anything else